docs/develop/create-sso-office-add-ins-nodejs.md

```diff
@@ -47,7 +47,9 @@ This article walks you through the process of enabling single sign-on (SSO) in a
 
 1. Run the command `npm run install-dev-certs`. Select **Yes** to the prompt to install the certificate.
 
-## Register the add-in with Azure AD v2.0 endpoint
+## Register the add-in's middle-tier web server with Azure AD v2.0 endpoint
+
+You need to create an app registration in Azure that represents your web server's middle tier. This will enable authentication support so that proper access tokens can be issued to the client code in JavaScript.
```

Comment:
Could you add a sentence here explaining what a middle-tier web server is, and why it's needed? The rest of the article refers to a middle-tier server but it's not clear to me what that is.

Reply:
Added explanation in overview.